### PR TITLE
[BERT-NER] Document model support (to release branch)

### PR DESCRIPTION
**Pull request by yekruglov**
Ticket: 46650

PR to master branch: https://github.com/openvinotoolkit/openvino/pull/6238

### CODE REVIEW

**Review by vgavrilo — APPROVED** [review]

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                  opset_version=11,
+                  )
+```
+
+The script generates ONNX* model file `bert-ner.onnx`.
+
+## Convert ONNX* BERT-NER model to IR
+
+```bash
+python mo.py --input_model bert-ner.onnx --input "input_mask[1 128],segment_ids[1 128],input_ids[1 128]"
```
[the remark this answers is not in the document]
Comment (yekruglov) [reply docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md:53]: Not sure how it should look
Is this enough?

[the remark this answers is not in the document]
Comment (lazarevevgeny) [reply docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md:53]: Good

**Review by lazarevevgeny — APPROVED** [review]

**Comment by yekruglov** [comment]
@openvinotoolkit/openvino-docs-maintainers Can anybody review this PR, please?

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Convert PyTorch* BERT-NER to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Bert_ner}
+
+## Download and Convert the Model to ONNX*
+
+To download a pre-trained model or train the model yourself, refer
+to the [instruction](https://github.com/kamalkraj/BERT-NER/blob/dev/README.md) in the
+BERT-NER model repository. The model with config files is stored in the directory with name `out_base`.
```
Comment (andrew-zaytsev) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md:7]: in the directory with name `out_base` --> in the `out_base` directory.

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+To convert the model to ONNX* format create and run the script with the following content in the root
```
Comment (andrew-zaytsev) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md:9]: To convert the model to ONNX* format create and run the script with the following content in the root
directory of the model repository. If you download the pre-trained model, you will need
to download [`bert.py`](https://github.com/kamalkraj/BERT-NER/blob/dev/bert.py) to run the script.-->

To convert the model to the ONNX* format, create and run the script with the following content in the root
directory of the model repository. If you download the pre-trained model, you need
to download [`bert.py`](https://github.com/kamalkraj/BERT-NER/blob/dev/bert.py) to run the script.

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+directory of the model repository. If you download the pre-trained model, you will need
+to download [`bert.py`](https://github.com/kamalkraj/BERT-NER/blob/dev/bert.py) to run the script.
+The instruction was tested with repository hash commit `e5be564156f194f1becb0d82aeaf6e762d9eb9ed`.
```
Comment (andrew-zaytsev) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md:12]: with repository hash commit --> with the repository hash commit

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**